x-pack/platform/plugins/shared/task_manager/README.md

```diff
@@ -616,20 +639,67 @@ The task manager's public API is create / delete / list. Updates aren't directly
 
 - Unit tests:
 
-   Documentation: https://www.elastic.co/guide/en/kibana/current/development-tests.html#_unit_testing
+  Documentation: https://www.elastic.co/guide/en/kibana/current/development-tests.html#_unit_testing
+
+  ```
+  yarn test:jest x-pack/platform/plugins/shared/task_manager --watch
+  ```
 
-   ```
-   yarn test:jest x-pack/platform/plugins/shared/task_manager --watch
-   ```
 - Integration tests:
-   ```
-   node scripts/functional_tests_server.js --config x-pack/test/plugin_api_integration/config.ts
-   node scripts/functional_test_runner --config x-pack/test/plugin_api_integration/config.ts
-   ```
+  ```
+  node scripts/functional_tests_server.js --config x-pack/test/plugin_api_integration/config.ts
+  node scripts/functional_test_runner --config x-pack/test/plugin_api_integration/config.ts
+  ```
 
 ## Monitoring
 
 Task Manager exposes runtime statistics which enable basic observability into its inner workings and makes it possible to monitor the system from external services.
 
 Public Documentation: https://www.elastic.co/guide/en/kibana/master/task-manager-health-monitoring.html
-Developer Documentation: [./MONITORING](./MONITORING.MD)
\ No newline at end of file
+Developer Documentation: [./MONITORING](./MONITORING.MD)
+
+### Schedule options
+
+We keep the scheduling config under the schedule field.
+And there are 2 different config options for scheduling a task:
+
+- `schedule.interval`
+  This is a basic interval string such as `1h`,`3m` or `7d` etc.
+
+- `schedule.rrule`
+  This is a subset of the rrule library.
+  We support only daily, weekly and monthly schedules so far.
+
+Monthly schedule options:
```

Comment:
Can you provide some examples? For example, every Sunday at 5pm, third Tuesday of every month at 1pm, every day at 10:05am

Reply:
Added some examples.
But unfortunately `every third Tuesday` is not possible with our rrule library.
In the original lib, `byweekday` is a class and has `.nth()` feature, but in ours it is just an enum.